dist/index.html

```diff
@@ -14,21 +14,16 @@
          <h1 class="text-center">Weather App</h1>
          <form class="search-location mb-4">
               <label for="city">Enter a city name</label>
-               <input type="text" name="city" class="form-control">
+              <input type="text" name="city" class="form-control">
          </form>
```

Comment:
### [Optional]
It would be nice to attach a submit button to this form, so the user can click the search instead of enter button on the PC